docs/docs/behaviors/macros.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Macro Behavior
+sidebar_label: Macros
+---
+
+## Summary
+
+The macro behavior allows configuring a sequence of other behaviors to invoke
+when the macro is pressed, including support for completing part of the sequence
+once the macro is released.
+
+## Macro Definition
+
+Each macro you want to use in your keymap gets defined first, then bound in your keymap.
+
+A macro definition looks like:
+
+```
+/ {
+    behaviors {
+        alt_tab: alt_tab_macro {
```

Comment:
Since I know this stuff was basically magic to me before I started to understand DeviceTree, it may be helpful to clarify somewhere (in a comment inline in the code?) that the part before the colon is the label you will use in keymap bindings, and the part after the colon doesn't matter as long as it's unique.

(That suggestion also applies to the other docs pages like hold-tap and mod-morph.)